integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -45,7 +45,7 @@
 
 small_data = [(1, 1.0), (1, 2.0), (2, 3.0), (2, 5.0), (2, 10.0)]
 
-large_data = list(map(lambda i: (i, i/1.0), range(1, 5000))) * 2
+large_data = list(map(lambda i: (i, i/1.0), range(1, 512))) * 2
```

Comment:
This change appears to be unrelated to running UDFs with an empty partition.  Is there some reason I am missing?

Reply:
Updated the PR description for this change, I just explained it in the commit message only before. 
This change is to save the cudf tests execution time, especially for databricks, and it is unrelated to the IPC error.